mmv1/products/cloudrun/api.yaml

```diff
@@ -616,6 +616,68 @@ objects:
                     required: true
                     description: |-
                       This must match the Name of a Volume.
+            - !ruby/object:Api::Type::NestedObject
+              name: startupProbe
+              min_version: beta
+              description: |-
+                Startup probe of application within the container.
+                All other probes are disabled if a startup probe is provided, until it
+                succeeds. Container will not be added to service endpoints if the probe fails.
+              properties:
+                - !ruby/object:Api::Type::Integer
+                  name: initialDelaySeconds 
+                  description: |-
+                    Number of seconds after the container has started before the probe is
+                    initiated.
+                    Defaults to 0 seconds. Minimum value is 0. Maximum value is 240.
```

Comment:
I see that for these integer fields, you've set them as `default_from_api` rather than setting clientside defaults. Is that for a specific reason? Assuming these values are fixed in the API, Terraform-local defaults would be preferable.

See https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/README.md#terraformyaml for details on `default_from_api`. It's a good solution when necessary, but pretty heavyweight- local defaults are significantly better when possible.

Reply:
Didn't know that `default_value` is available. Done.